### ISSUE TITLE
Retinal drusen

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#10955]
Preferred term label:
Retinal drusen

Synonyms
Drusen of the retina, Retinal deposits, Hard drusen, Soft drusen

Definition (free text, please give PubMed ID)
Although we already have macular drusen (30499), not all drusen are in the macula.

Parent term (use hpo.jax.org/app)
Abnormality of the retina (HP:0000479)

Diseases characterized by this term
Age-related macular degeneration

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
@daboe01 
Adding new term: 
- ID: HP:6001388
- label: Retinal drusen
- parents: [Abnormal retinal morphology (HP:0000479)](https://hpo.jax.org/app/browse/term/HP:0000479)
- pmid: PMID:19726434|PMID:30404862
- def: Small focal extracellular deposits underneath the retina, visible ophthalmoscopically as yellow dots.
- comment: Drusen are lipid-, mineral-, and protein-containing extracellular deposits that accumulate between the basal lamina of the retinal pigment epithelium and Bruch's membrane of the human eye. They are a defining feature of age-related macular degeneration, a common sight-threatening disease of older adults.
- orcid: orcid:0000-0001-9042-0508
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10955
- synonyms:
